test/integration/node-specific/client_close.test.ts

```diff
@@ -452,30 +457,72 @@ describe.skip('MongoClient.close() Integration', () => {
     });
   });
 
-  describe('ClientSession (Implicit)', () => {
-    describe('Server resource: LSID/ServerSession', () => {
-      describe('after a clientSession is implicitly created and used', () => {
```

Comment:
We still want this test for implicit sessions, right?  Just not the transaction tests because you cannot run a transaction on an implicit session.

Reply:
I don't quite understand.  Here's my thought - I'm imagining a test that 1. creates an implicit session (i.e., runs an operation) 2. confirms it exists server-side 3. closes the client 4. confirms it doesn't exist server side.

Which part in particular do you think isn't possible / valuable?

Reply:
From slack: we reviewed the test cases from the document and seem to have the coverage we planned on. Correct?